Kernel/SortHelper.cpp

```diff
@@ -361,8 +361,7 @@ void SortHelper::collectVariableSortsIter(CollectTask task, DHMap<unsigned,TermL
     
         unsigned position = 0;
         for (TermList* ts = term->args(); ts->isNonEmpty(); ts = ts->next()) {
-          CollectTask newTask;
```

Comment:
The linter doesn't like that the enumeration `CollectTask::fncTag` is left unitialised by the constructor and filled in later. I'm not sure what the standard says about this, but in this case there's no need for it - we always know at `CollectTask()` what the `fncTag` will be.

Reply:
Nice! This also looks like a neater design anyway!